### ISSUE TITLE
The npm section is hard to follow

### ISSUE THREAD

**Issue by mikeboutros** [freeCodeCamp/freeCodeCamp#36610]
**Describe your problem and how to reproduce it:**
The entire curriculum has issues with grammar and the landing page where it explains how to set up the environment is confusing.

**Add a Link to the page with the problem:**
The entire curriculum: https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm

**Tell us about your browser and operating system:**
* Browser Name: Firefox
* Browser Version: 68.0.1
* Operating System: macOS Mojave 

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

**Comment by RandellDawson** [comment]
@mikeboutros Can you be a bit more specific as to what needs to be changed in your opinion.  Make sure to review the current version of the challenges on the master branch, as some of the verbiage could have changed since the production site was last updated.

Please make your recommended changes here and we can discuss.  If a consensus is reached about what should be changed, we can request others help create one or more pull requests to improve the challenges.

**Comment by mikeboutros** [comment]
[Introduction to the Managing Packages with npm Challenges](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm)

On this page, we need to explain that if you log into Github on Glitch, you have to do certain steps. Something similar to what this person [said](https://www.freecodecamp.org/forum/t/managing-packages-with-npm-how-to-use-package-json-the-core-of-any-node-js-project-or-npm-package/198736/13). I found it difficult to go to the next step without first understanding how to get the link.

[How to Use package.json, the Core of Any Node.js Project or npm Package](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/how-to-use-package-json-the-core-of-any-node-js-project-or-npm-package)

> Add your name to the author-field in the package.json of your Glitch project.

I didn't have a prefilled or empty author field. I had to make one which isn't explained in the instructions. This applies to all pages as none of these are default in a Glitch project.

[Add a Description to Your package.json](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/add-a-description-to-your-package-json)

Grammatical errors (see bold):

> If you some day plan to **publishing a package to npm**, remember that this is the string that should sell your idea to the user when they decide whether to install your package or not. However, that’s not the only use case for **the description: Since it’s** a great way to summarize what a project does, it’s just as important for your normal Node.js-projects to help other developers, future maintainers or even your future self understand the project quickly.

Other issues:

>Instructions
Add a description to the package.json in your Glitch project.

It is unclear where to put the description.

[Managing Packages with Npm - Add a License to Your package.json](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/add-a-license-to-your-package-json)

> http://choosealicense.com is a great resource...

Should be a hyperlink.

[Managing Packages with Npm - Add a Version to Your package.json](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/add-a-version-to-your-package-json)

> The version is together with name one of the required fields in a package.json.

Needs to be rephrased.

[Managing Packages with Npm - Expand Your Project with External Packages from npm](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/expand-your-project-with-external-packages-from-npm)

> Add version 2.14.0 of the package moment to the dependencies-field of your package.json

The average user will not understand what it is being asked here. May need to be rephrased.

[Managing Packages with Npm - Use the Tilde-Character to Always Use the Latest Patch Version of a Dependency](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/use-the-tilde-character-to-always-use-the-latest-patch-version-of-a-dependency/)

> Note that the version numbers themselves not should be changed.

Needs to be rephrased.

[Managing Packages with Npm - Use the Caret-Character to Use the Latest Minor Version of a Dependency](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/use-the-caret-character-to-use-the-latest-minor-version-of-a-dependency/)

> Similar to how the tilde (~) we learned about...
> Note that the version numbers themselves not should be changed.

Needs to be rephrased.

Grammatical error:

>PATCHes

[Managing Packages with Npm - Remove a Package from Your Dependencies](https://learn.freecodecamp.org/apis-and-microservices/managing-packages-with-npm/remove-a-package-from-your-dependencies/)

> Now you’ve tested a few ways you can manage dependencies of your project by using the package.json's dependencies-section. You’ve included external packages by adding them to the file and even told npm what types of versions you want by using special characters as the tilde (~) or the caret (^).

I would reword this with something like:

> Now that you have tested a few ways to manage dependencies in your project, adding external packages by adding them to the package.json file, and even told npm what types of versions you want by using special characters like the tilde (~) and/or the caret (^), what if you wanted to remove a package?

**Comment by moT01** [comment]
Closing as stale. Thanks and happy coding 🎉
